pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -433,6 +437,7 @@ func (r *ReconcileHyperConverged) ensureHco(req *hcoRequest) error {
 			})
 			return err
 		}
+		req.componentUpgradeDone = req.componentUpgradeDone && upgradeDone
```

Comment:
not really sure about this:
it was initialized to true, we set it to false on the first component that is still not upgrade.
When do we set it back to true?